cql3/restrictions/single_column_primary_key_restrictions.hh

```diff
@@ -187,6 +187,7 @@ public:
                             join(", ", get_column_defs())));
         }
         do_merge_with(::static_pointer_cast<single_column_restriction>(restriction));
+        this->expression = make_conjunction(std::move(this->expression), restriction->expression);
```

Comment:
I looked a little at callers and I think we're safe with std::move() the rhs too. All the callers of add_restriction() (which eventually call merge_with()) call in with a temporary restriction.

But I'm find with delaying std::move() until later.

Reply:
Yeah, those callers will change in the very next step of the grand plan.  I promise I'll add std::move()s everywhere when the code settles.